zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServerMain.java

```diff
@@ -163,7 +163,9 @@ public void runFromConfig(ServerConfig config) throws IOException, AdminServerEx
                 zkServer.getZKDatabase(),
                 zkServer.firstProcessor,
                 Integer.getInteger("znode.container.checkIntervalMs", (int) TimeUnit.MINUTES.toMillis(1)),
-                Integer.getInteger("znode.container.maxPerMinute", 10000));
+                Integer.getInteger("znode.container.maxPerMinute", 10000),
+                Long.getLong("znode.container.maxNeverUsedIntervalMs", Long.MAX_VALUE)
```

Comment:
what about using zero a value for 'disabled'?
It is very common to use zero as special value in order to 'not enable' features

Reply:
OK with me - or maybe `-1`?

Reply:
Personally I would go for 0. Not so important.